src/Components/Header/Header.scss

```diff
@@ -0,0 +1,179 @@
+@import url('https://fonts.googleapis.com/css2?family=Bitter:wght@800&family=Bitter:wght@500&family=Kalam&family=Kaushan+Script&display=swap');
+
+.header {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  height: auto;
+  background-image: url(../../../public/img/header-darken.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center 44%;
+  transition: all 1s;
+  -webkit-transition: all 1s;
+  -moz-transition: all 1s;
+  -ms-transition: all 1s;
+  -o-transition: all 1s;
+  padding-top: 3%;
+  padding-bottom: 5%;
+  color: #fff;
+
+  &__title {
+    line-height: 0%;
+    text-align: center;
+    margin: 0 auto;
+    font-family: 'Bitter', serif;
+    font-size: 1rem;
+
+    &-description {
+      line-height: 80%;
+    }
+
+    h1 {
+      font-size: 2rem;
+
+      span {
+        font-family: 'Kalam', cursive;
+        font-size: 2.4rem;
+        font-weight: 300;
+      }
+    }
+
+    hr {
+      width: 25%;
+      background-color: #fff;
+    }
+  }
+
+  &__logo {
+    margin-top: 0.1rem;
+    margin-left: 10%;
+    line-height: 100%;
+
+    svg {
+      transition: all 1s;
+      -webkit-transition: all 1s;
+      -moz-transition: all 1s;
+      -ms-transition: all 1s;
+      -o-transition: all 1s;
+      width: 35px;
```

Comment:
What do you think, if width and height would be much bigger than it is set now? I think, that this size is nice and fits to hamburger's size, but as logo, my eyes want it be larger.

Reply:
Agree, I increased it for wide screen displays (increased till 55px width and height accordingly), for smaller devices it may stay the same as is. That's one more reason to create Header's StoryBook, just to give others possibility to evaluate live preview ;).

Reply:
Perfect, thanks!